dsub_beast.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# This is only a demonstration of how to run beast via dsub. 
+# The accelerator (GPU) type and count should be set according to 
+# the size of the data and number of partitions.
+
+# --accelerator-count should scale with number of partitions in data
+# --nvidia-driver-version must match compatible CUDA version
+# 
+
+function print_usage(){
+    echo "Usage: $(basename $SCRIPT) gs://path/to/in.xml gcp-project-name"
+}
+
+if [ $# -eq 0 ]; then
+    print_usage
+    exit 1
+fi
+
+IN_XML="$1"
+OUT_BUCKET="$(dirname $1)"
+GCP_PROJECT="$2"
+DOCKER_IMAGE="quay.io/broadinstitute/beast-beagle-cuda"
+
+dsub \
+  --provider=google-v2 \
+  --project "${GCP_PROJECT}" \
+  --zone "us*" \
+  --accelerator-type "nvidia-tesla-k80" \
+  --nvidia-driver-version "396.37" \
+  --accelerator-count 1 \
+  --image "${DOCKER_IMAGE}" \
+  --input "INPUT_FILE=${IN_XML}" \
+  --output "OUTPUT_FILES=${OUT_BUCKET}/*" \
+  --logging "${OUT_BUCKET}" \
+  --script 'run_beast.sh' \
+  --boot-disk-size 30 \
```

Comment:
I wonder if `boot-disk-size` doesn't need to be so big? It depends mostly on how big this nvidia base layer is to begin with, but it's not like we're adding conda environments on top of it.

Reply:
The cuda base and runtime images are reasonably compact, but in order to build beagle we need the full cuda build toolchain which takes up a fair amount of space. Calling `df-h` on the finished image tells me 4.5G is used, but I was seeing "no space left on device" errors with the default boot disk size of 10G to I bumped it up to 30G, but we can probably throttle it back.